cli/cli/helpers/oauth/oauth.go

```diff
@@ -40,7 +40,7 @@ type OAuth interface {
 func AuthFlow() (string, string, error) {
 	httpClient := &http.Client{} // nolint: exhaustruct
 
-	minimumScopes := []string{"repo", "read:org", "gist"}
+	minimumScopes := []string{"repo"}
```

Comment:
can this be repo:read?

Reply:
No `repo:read` scope exists :( a la: https://kurtosistech.slack.com/archives/D040G0YT5G8/p1708451376845819)
https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/scopes-for-oauth-apps